pkg/cmd/task/describe.go

```diff
@@ -0,0 +1,174 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import (
+	"fmt"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const describeTemplate = `Name:	{{ .Task.Name }}
+Namespace:	{{ .Task.Namespace }}
+
+Input Resources
+{{- if not .Task.Spec.Inputs }}
+No input resources
+{{- else }}
+{{- if eq (len .Task.Spec.Inputs.Resources) 0 }}
+No input resources
+{{- else }}
+NAME	TYPE
+{{- range $ir := .Task.Spec.Inputs.Resources }}
+{{ $ir.Name }}	{{ $ir.Type }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+Output Resources
```

Comment:
Can we have a colon i.e: 

Output Resources:

and two space after for the type, ie same as  what we have on `oc describe`: 

![image](https://user-images.githubusercontent.com/98980/65586144-f1166f80-df83-11e9-82ac-283f7b6fd73b.png)

(not just for output resources but for the others too)

Reply:
@chmouel 

If you see any describe command i.e. pipelinerun, taskrun, pipeline, etc. All are in the same pattern. I made it consistent throughout